src/account/accountsReducer.js

```diff
@@ -32,6 +33,7 @@ const realmAdd = (state, action) => {
       apiKey: '',
       email: '',
       ackedPushToken: null,
+      zulipFeatureLevel: action.zulipFeatureLevel,
```

Comment:
An intermediate commit makes this always `null`, and a fixup commit at the end fills in the proper logic. Instead it can get the proper logic to begin with -- just squash the fixup commit into the other one.